files/default/sql/ddl/updates/undo/3.2.0__undo.sql

```diff
@@ -66,3 +66,21 @@ DROP TABLE IF EXISTS `hopsworks`.`monitoring_window_config`;
 DROP TABLE IF EXISTS `hopsworks`.`statistics_comparison_config`;
 
 DROP TABLE IF EXISTS `hopsworks`.`job_schedule`;
+-- alert feature monitoring changes
+DROP TABLE IF EXISTS `hopworks`.`feature_store_alert`;
```

Comment:
I don't think you can do that, you would lose all the data in the table if you drop and create.
You can delete all entries which have feature_view_id not null and then modify the table by dropping columns